bootstrap.bash

```diff
@@ -89,6 +83,7 @@ BOB_DIR="$(relative_path $(pwd) "${SCRIPT_DIR}")"
 CONFIG_FILE="${BUILDDIR}/${CONFIGNAME}"
 CONFIG_JSON="${BUILDDIR}/config.json"
 
+export TOPNAME="build.bp"
```

Comment:
Please update the BOOTSTRAP_VERSION in this commit, as this causes an incompatibility.